### PR TITLE
Object Detection

### LINKED ISSUE #199: Object Detection
Hey, I'd like to work on Object Detection under the Artificial Intelligence Domain for GSSoC'22

### PR DESCRIPTION
**Pull request by Jatin7385**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

This is an Object Detection project for Images. The project uses ssd_mobilenet pretrained model, OpenCV and Matplotlib for Object detection in images. 

Fixes #199 

### Checklist

- [ ✓] I've read the contribution guidelines.
- [ ✓] I've checked the issue list before deciding what to submit.
- [ ✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
None

### CODE REVIEW

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍 
@photon149 Please check and if green flag from your side then tag other mentor .

File: Artificial Intelligence/Basic/Object_Detection/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Object Detection
+This is a basic Object Detection project that uses ssd_mobilenet pretrained model, OpenCV and Matplotlib for Object detection of images.
```
Comment (prakhar3062) [inline Artificial Intelligence/Basic/Object_Detection/README.md:2]: If possible can you add more description of your project in readme file

Comment (Jatin7385) [reply Artificial Intelligence/Basic/Object_Detection/README.md:2]: Alright will do

**Comment by Jatin7385** [comment]
@prakhar3062 I have added more content to the readme file. Hope it is fine now.

**Review by prakhar3062 — APPROVED** [review]
@photon149 okay from my side

**Review by photon149 — APPROVED** [review]
Approved from my end.
@Kushal997-das please merge the same , if no conflicts.

**Pull request merged**